graviti/dataframe/column/series.py

```diff
@@ -208,6 +208,24 @@ def _get_repr_indices(self) -> Iterable[int]:
     def _del_item_by_location(self, key: Union[int, slice]) -> None:
         self._data.__delitem__(key)
 
+    @classmethod
```

Comment:
maybe remove the "support" in the commit message is better.
```
feat(dataframe): refresh the data of dataframe after uploading
```
`support` sounds like the the new feature is optional, but here this refresh feature is not an optional feature.